packages/react-components/src/components/FeedbackDialog/types.ts

```diff
@@ -0,0 +1,13 @@
+export type Category =
```

Comment:
is this file used anywhere?

Reply:
Good catch, not needed, as it's part of the API!